### PR TITLE
Iterator on teacher panel

### PR DESCRIPTION
**Pull request by dmcavoy**
Adds iterator next to the student information at the top of the teacher panel so you can cycle through the list of students. Will now show the teacher information at the top of the panel when "Me" is clicked. 

Screen Shot 2019-06-07 at 2 48 45 PM
Screen Shot 2019-06-07 at 3 07 27 PM

### CODE REVIEW

**Review by davidsbailey — APPROVED** [review]
This looks great, Dani!

UX question: do we want to gray out the arrows when there is no previous/next, so the teacher can avoid accidentally cycling through the students a second time? apologies if this is addressed in a spec somewhere.

Other question: commit description says stage extras is "mostly working". can you elaborate briefly?

File: apps/src/code-studio/components/progress/ScriptTeacherPanel.jsx [hunk 1]
```diff
@@ -95,10 +95,18 @@ class ScriptTeacherPanel extends React.Component {
...
+      if (currentStudent) {
+        if (currentSectionScriptLevels) {
+          currentStudentScriptLevel = currentSectionScriptLevels.find(
+            level => this.props.getSelectedUserId() === level.user_id
+          );
+        }
+      } else {
+        currentStudent = {
+          id: null,
+          name: 'Me'
```
Comment (maddiedierker) [inline apps/src/code-studio/components/progress/ScriptTeacherPanel.jsx]: nit: use an i18n string here

File: apps/src/code-studio/components/progress/SelectedStudentInfo.jsx [hunk 2]
```diff
@@ -62,67 +82,107 @@ export class SelectedStudentInfo extends React.Component {
...
+                text={i18n.unsubmit()}
+                color="blue"
+                onClick={this.onUnsubmit}
+                id="unsubmit-button-uitest"
+                disabled={level.status !== LevelStatus.submitted}
+              />
+            </div>
+          )}
+          {level.contained && (
+            <div>
```
Comment (maddiedierker) [inline apps/src/code-studio/components/progress/SelectedStudentInfo.jsx]: nit: remove extra div (unless it's needed for a specific reason)

**Review by maddiedierker — APPROVED** [review]
🎉 really great work and refactoring here!

**Comment by dmcavoy** [comment]
@davidsbailey I think we actually want to be able to continuously cycle through the names - when you get to the end go back to the beginning. At least that was my understanding of the spec.

In terms of the "mostly working" on the commit history. That was from before I figured out that that one level was not working because it didn't get a User Level which I don't think is connected to the changes that are in this branch but just exposed by them.

File: apps/src/code-studio/components/progress/SelectedStudentInfo.jsx (hunk 2 above)
Comment (dmcavoy) [reply apps/src/code-studio/components/progress/SelectedStudentInfo.jsx]: the inner conditionals that determine showing the navigator and driver info don't like it if you remove the outer div

**Pull request merged**